src/AWS.Deploy.Orchestration/ServiceHandlers/AWSElasticBeanstalkHandler.cs

```diff
@@ -204,6 +212,65 @@ public void SetupWindowsDeploymentManifest(Recommendation recommendation, string
             }
         }
 
+        /// <summary>
+        /// When deploy a self contained deployment bundle Beanstalk needs a Procfile to tell the environment what process to start up.
```

Comment:
replace "deploy" by "deploying" and add a comma after "bundle"

Reply:
Done